Warning: Calling depends_on :x11 is deprecated! Use depends_on specific X11 formula(e) instead.

MacOS Big Sur
Titel is the warning during the upgrade.

If I try to reinstall it, an error message is

> Last 15 lines from /Users/jomic/Library/Logs/Homebrew/xfig/02.make:
In file included from d_line.c:18:
./fig.h:193:10: fatal error: 'X11/Xlib.h' file not found
#include <X11/Xlib.h>
         ^~~~~~~~~~~~
1 error generated.
make[1]: *** [d_ellipse.o] Error 1
1 error generated.
1 error generated.
1 error generated.
make[1]: *** [d_regpoly.o] Error 1
make[1]: *** [d_picobj.o] Error 1
make[1]: *** [d_spline.o] Error 1
1 error generated.
make[1]: *** [d_line.o] Error 1
make: *** [install-recursive] Error 1

> If reporting this issue please do so at (not Homebrew/brew or Homebrew/core):
  https://github.com/mistydemeo/homebrew-xfig/issues

--- a/xfig.rb
+++ b/xfig.rb
@@ -3,11 +3,12 @@ class Xfig < Formula
   homepage "https://mcj.sourceforge.io"
   url "https://downloads.sourceforge.net/project/mcj/xfig-3.2.7a.tar.xz"
   sha256 "ca89986fc9ddb9f3c5a4f6f70e5423f98e2f33f5528a9d577fb05bbcc07ddf24"
+  revision 1
 
   depends_on "fig2dev"
   depends_on "ghostscript"
   depends_on "jpeg"
-  depends_on :x11
+  depends_on "libxaw3d"
 
   def install
     system "./configure", "--prefix=#{prefix}",

